Kitodo-PersistentIdentifier/src/test/java/org/kitodo/persistentidentifier/URNPersistentIdentifierTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.persistentidentifier;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests for PersistentIdentifier.
+ */
+public class URNPersistentIdentifierTest {
+
+    @Test
+    public void shouldGenerateUnifiedResourceName() throws Exception {
+        URNPersistentIdentifier urnPersistentIdentifier = new URNPersistentIdentifier();
+
+        String urn = urnPersistentIdentifier.generateUnifiedResourceName("bsz", "14", "db", "378704842");
```

Comment:
Missing `id` in last parameter as `id` is part of this used identifier value! This must be corrected for other tests too.